library/alloc/src/string.rs

```diff
@@ -1388,6 +1388,7 @@ impl String {
     /// assert_eq!(fancy_f.len(), 4);
     /// assert_eq!(fancy_f.chars().count(), 3);
     /// ```
+    #[doc(alias = "length")]
```

Comment:
Do you think an additional `bytes_len` would be good here since it reflects that? I am not sure if anyone would type that out, if we put it it may be hidden.

Reply:
Rust documentation system doesn't render aliases unless you are directly searching for them, and I don't think anyone would type `bytes_len` specifically.